department-of-veterans-affairs/vets-website
# Add unreported assets chapter

## Did you change site-wide styles, platform utilities or other infrastructure?

## Summary
Import Assets Previously not Reported - Section 10 into the Income & Asset Statement form (21P-0969)

## Issue
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/86448

## Associated Pull Request(s)
n/a

## How to run in local environment
1. Check out this branch locally
2. Run `vets-website` and `vets-api`
3. In your local environment, set the `income_and_assets_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/income_and_assets_form_enabled
4. Go to `http://localhost:3001/pension/apply-for-veteran-pension-form-21p-527ez/` in your browser

## How to verify
1. Go to chapter 10 - Assets previously not reported (/unreported-assets-summary)
2. Verify that an unreported asset can be added, edited, and deleted
3. Verify that unreported assets are displayed on the 'Review' page

## What areas of the site does it impact?
Income & Asset Statement Application

## Screenshots
### Desktop
#### Summary Page
| No Assets      | Assets       |
| ----------- | ----------- |
|![localhost_3001_income-and-asset-statement-form-21p-0969_unreported-assets-summary](https://github.com/user-attachments/assets/b8cacfac-f3cb-4cd7-9cac-25d7932c0d0e)|![localhost_3001_income-and-asset-statement-form-21p-0969_unreported-assets_0_asset-type_add=true](https://github.com/user-attachments/assets/088f52b7-23c1-4dd0-8cd1-26e9a97e55c1)

### Item Pages
| Asset Relationship | Asset Type |
| ----------- | ----------- |
|![localhost_3001_income-and-asset-statement-form-21p-0969_unreported-assets-summary (1)](https://github.com/user-attachments/assets/c3711191-1893-49dc-acdc-b1d613cb5bd9)|![localhost_3001_income-and-asset-statement-form-21p-0969_unreported-assets-summary (2)](https://github.com/user-attachments/assets/bdd1c686-db6b-4be9-984b-9baba73bb2c7)


### Quality Assurance & Testing

### Error Handling

### Authentication

## Requested Feedback
(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
